asterius/src/Asterius/GHCi/Internals.hs

```diff
@@ -315,12 +315,20 @@ asteriusRunTH hsc_env _ _ q ty _ s ahc_dist_input =
   withTempDir "asdf" $ \tmp_dir -> do
     let p = tmp_dir </> "asdf"
     distNonMain p [runner_sym, run_mod_fin_sym] ahc_dist_input
+    rts_val <- importMJS s $ p `replaceFileName` "rts.mjs"
     mod_buf <- LBS.readFile $ p -<.> "wasm"
-    lib_val <- importMJS s $ p -<.> "lib.mjs"
-    f_val <- eval s $ takeJSVal lib_val <> ".default"
+    req_mod_val <- importMJS s $ p -<.> "req.mjs"
+    req_val <- eval s $ takeJSVal req_mod_val <> ".default"
     buf_val <- alloc s mod_buf
     mod_val <- eval s $ "WebAssembly.compile(" <> takeJSVal buf_val <> ")"
```

Comment:
Streaming compilation API is available only in browsers, not `node`.

As for `instantiateStreaming`, even in browsers we can't use that right now, since `newAsteriusInstance` expects an already compiled module to be a part of input; we can change that if needed but that should be in a separate PR.

Reply:
> Streaming compilation API is available only in browsers, not node.

Oh, ok :/